cypher-shell/src/main/java/org/neo4j/shell/ShellRunner.java

```diff
@@ -1,10 +1,6 @@
 package org.neo4j.shell;
 
-import org.neo4j.shell.cli.CliArgHelper;
-import org.neo4j.shell.cli.FileHistorian;
-import org.neo4j.shell.cli.InteractiveShellRunner;
-import org.neo4j.shell.cli.NonInteractiveShellRunner;
-import org.neo4j.shell.cli.StringShellRunner;
+import org.neo4j.shell.cli.*;
```

Comment:
Do you have strong opinions on importing `*` from a package? I am strongly in favor of always requiring explicit imports (this requires a change in intellij settings)